modules/test/playwright/tests/frontend-data-set-views-web/pages/DataSetsPage.ts

```diff
@@ -60,20 +60,23 @@ export class DataSetsPage {
 		await this.newDataSetModal.nameInput.fill('Data Set Sample');
 		await this.newDataSetModal.restApplicationField.click();
 		await this.newDataSetModal.restApplicationOptions
-			.getByRole('option', {name: '/data-set-manager/fields'})
+			.getByRole('option', {name: '/headless-delivery/v1.0'})
```

Comment:
Soon, we'll need to parameterize this in order to create specific data sets to test separate scenarios. Ideally, we shall base on well-known API endpoints/schemas, for instance, the ones representing DSM entity persistence. As long as they reflect the complexity we need, this looks a sensible choice.

We can move on with this though

Reply:
DataSet parametrization will be available with the actions [PR](https://github.com/liferay-frontend/liferay-portal/pull/3978/files#diff-79653e196ebf0913d3d203ed9cec87f1aaa39fce6e1ad5a82e5f495142d8e6dbR72)

Reply:
Sure, but we must take into account that _all_ data used in the whole chain of cases must be related. That said, the endpoint we use here comes with different fields we must take care about in subsequent places (may be a good idea to create a big and consistent bundle of test data).